pkgs/development/python-modules/deap/default.nix

```diff
@@ -9,19 +16,19 @@ buildPythonPackage rec {
     hash = "sha256-zAHemJLfp9G8mAPasoiS/q0XfwGCyB20c2CiQOrXeP8=";
   };
 
-  propagatedBuildInputs = [ numpy matplotlib ];
+  propagatedBuildInputs = [ matplotlib numpy ];
+  nativeCheckInputs = [ pytestCheckHook ];
 
-  nativeCheckInputs = [ nose ];
   checkPhase = ''
-    nosetests --verbosity=3
+    runHook preCheck
+    ${python3.interpreter} -m unittest
+    runHook postCheck
   '';
```

Comment:
I think we can remove this `checkPhase` since we use the provided one by pytestCheckHook.

Reply:
Great!